pkg/processor/metricfrequencyprocessor/config.go

```diff
@@ -0,0 +1,22 @@
+package metricfrequencyprocessor
+
+import (
+	"time"
+
+	"go.opentelemetry.io/collector/config"
+)
+
+// Config defines configuration for Source processor.
```

Comment:
I don't believe this is a `Source` processor :)

Reply:
oops, fixed